nix/scripts.nix

```
@@ -12,7 +12,7 @@ let
   '';
   run = pkgs.writeShellScriptBin "run" ''
     set -euo pipefail
-    result/bin/replaceme
+    result/bin/replaceme $@
```

```suggestion
    result/bin/replaceme "$@"
```
(See: https://github.com/koalaman/shellcheck/wiki/SC2068)